### PR TITLE
Added ability to set uuid as item_id type while generating migration

### PR DESCRIPTION
**Pull request by Nuzair46**
Added option `--uuid` when generating new migration. This can be used to set the type on `item_id` column to uuid.

Use case: 

* If a user uses paper_trail on a database that uses `uuid` as primary key, there is a chance where `item_id` becomes `0` causing the versioning to not work. Changing the type of `item_id` from `bigint` to `string` fixes this issue.
* This PR makes it possible for the user to use `uuid` while generating the migration.

```
bin/rails generate paper_trail:install [--uuid]
```

Thank you for your contribution!

Check the following boxes:

- [x] Wrote [good commit messages][1].
- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [x] Added tests.
- [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new 
  code introduces user-observable changes.
- [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1152,176 +1152,177 @@ candidates.
...
-  - Updated deprecation warnings for `Model.paper_trail_on` and
-    `Model.paper_trail_off` to have display correct version number the methods
-    will be removed (`4.0`)
+- [#479](https://github.com/paper-trail-gem/paper_trail/issues/479) - Deprecated
+  `originator` method in favor of `paper_trail_originator` Deprecation warning
+  informs users that the `originator` of the methods will be removed in
+  version `4.0`. (Backported from v4)
+- Updated deprecation warnings for `Model.paper_trail_on` and
+  `Model.paper_trail_off` to have display correct version number the methods
+  will be removed (`4.0`)
```
Comment (jaredbeck) [inline CHANGELOG.md:1161]: Please do not change unrelated parts of the CHANGELOG. I'm guessing your IDE reformatted it automatically, but I don't want to read through 300 lines of diffs to confirm that.

File: spec/generators/paper_trail/install_generator_spec.rb [hunk 2]
```diff
@@ -102,4 +102,27 @@
...
     end
   end
+
+  describe "`--uuid` option set to `true`" do
+    before do
+      prepare_destination
+      run_generator %w[--uuid]
+    end
+
+    expected_item_id_type = "string"
```
Comment (jaredbeck) [inline spec/generators/paper_trail/install_generator_spec.rb:112]: Thanks for writing a test, but it's unusual IME to define a local variable in a `describe` block. Please move this variable into the `it` block.

**Review by jaredbeck — CHANGES_REQUESTED** [review]
Thanks Nuzair! Generally this looks good, just a few minor comments.

Also, please update the install documentation in the README.

**Comment by Nuzair46** [comment]
@jaredbeck , Suggested changes are made. Please review. Thanks.

closes #1373

**Comment by jaredbeck** [comment]
Thanks for the contribution!

**Pull request merged**